### PR TITLE
New Update Section in README.md

### PR DESCRIPTION
**Pull request by ahoglund**
Was updating my ruby-build version today and thought the README could be a bit clearer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,9 +40,21 @@ permission to `/usr/local`, you will need to run `sudo ./install.sh` instead.
...
 
+## Updating
+
 To update ruby-build after it has been installed, run `git pull` in your cloned
 copy of the repository, then re-run the install script.
 
+### For rbenv plugin
+    cd /path/to/.rbenv/plugins/ruby-build
+    git pull
+    ./install.sh
```
Comment (hsbt) [inline README.md]: this is redundant instruction.

Comment (mislav) [inline README.md]: Yes, if it's an rbenv plugin then it doesn't need running `./install.sh`.

**Comment by mislav** [comment]
Thanks for the suggestion, but I think we will stick with [the update information in the wiki](https://github.com/sstephenson/ruby-build/wiki#updating-ruby-build) for now. Updating ruby-build can be tricky for users who don't remember how they installed it (or have installed it twice, for that matter) so I'll try to come up with some tool that would automate this.

File: README.md (hunk 1 above)
Comment (ahoglund) [inline README.md]: thanks! didn't know that.

**Pull request closed without merging**